cmd/spire-agent/cli/run/run.go

```diff
@@ -43,6 +43,8 @@ const (
 	defaultLogLevel          = "INFO"
 	defaultDefaultSVIDName   = "default"
 	defaultDefaultBundleName = "ROOTCA"
+
+	maxTrustDomainLength = 255
```

Comment:
Could/should this be defined in [go-spiffe](https://github.com/spiffe/go-spiffe/blob/master/v2/spiffeid/trustdomain.go)?

Reply:
Fair.  I believe the const probably belongs in this repository though as SPIRE acts as a "URI producer" to use the RFC 3986 terminology. The functions in go-spiffe related to trust domains resolve around parsing it out of an existing string or `url.URL` and should therefore probably be lenient.

If that sounds reasonable, I'll update this PR to define this const only once in SPIRE.

Reply:
That sounds like the right solution. Thank you.

Reply:
Looks good